### DISCUSSION TITLE
V2.76

### DISCUSSION THREAD

**Discussion by zhongyang219** [zhongyang219/MusicPlayer2#508] [Announcements]
* 新增迷你模式自定义界面的功能，新增两种迷你模式界面。
* 新增自绘播放列表，界面4、界面6、Groove音乐风格、Groove音乐风格（Windows11）、默认界面新增自绘播放列表。
* 新增自定义专辑封面保存位置的功能。
* 新增ksc格式逐字歌词的支持。
* 新增播放列表右键菜单中的下一首播放的功能。
* 转换格式功能新增FLAC格式输出。
* 新增将正在播放曲目和播放记录同步到LastFM的功能。
* 新增触屏设备长按弹出右键菜单的支持。
* 新增将cue音轨保存到媒体库的功能。
* 新增自定义播放列表行高的功能。
* 新增自定义自绘标题栏按钮的功能。
* 新增深色/浅色切换按钮。
* 新增没有歌词时在歌词区域显示播放曲目信息的功能。
* 更新若干图标，修正部分图标在100%缩放比例时模糊的问题。
* 修正有对话框打开时无法通过通知区图标右键菜单退出程序的问题。
* 新增cue格式标签写入和显示高级标签信息的功能。
* 修正播放列表中含有cue音轨时保存为m3u和m3u8格式时不正常的问题。
* 媒体库中按音轨号排序时优先按CD序号排序。
* 媒体库更新时添加线程同步的处理，增加程序的稳定性。
* 修正转换格式时，转换出错仍然显示完成的问题。
* 媒体库中播放列表右键菜单中新增“修复错误的文件路径”和“打开播放列表所在位置”的功能。
* 新增记住用户自定义界面中StatackElement的序号的功能。
* 新增是否启用MediaTransControl的选项。
* 自定义界面中Text元素新增“艺术家 - 唱片集”类型。
* 其他细节方面的改进。
* 其他bug的修正。

**Comment by OmiChrome** [comment]
ENGLISH : 

- Added the function of customizing the mini-mode interface, and added two mini-mode interfaces.
- Added self-drawn playlist, interface 4, interface 6, Groove music style, Groove music style (Windows11), default interface added self-drawn playlist.
- Added the function of customizing the save location of the album cover.
- Added support for verbatim lyrics in ksc format.
- Added the function of playing the next song in the right-click menu of the playlist.
- Added FLAC format output to the format conversion function.
- Added the function of synchronizing the currently playing track and playing record to LastFM.
- Added support for long-pressing the pop-up right-click menu on touch-screen devices.
- Added the ability to save cue tracks to the media library.
- Added the ability to customize the row height of the playlist.
- Added the ability to customize self-drawn title bar buttons.
- Added dark/light toggle button.
- Added the function of displaying the playing track information in the lyrics area when there are no lyrics.
- Update several icons and fix the problem that some icons are blurry when zoomed in at 100%.
- Fixed the problem that the program cannot be exited through the right-click menu of the icon in the notification area when a dialog box is open.
- Added the function of writing cue format labels and displaying advanced label information.
- Fixed the problem that when the playlist contains cue tracks, it is not normal to save in m3u and m3u8 formats.
- When sorting by track number in the media library, sort by CD number first.
- When the media library is updated, thread synchronization processing is added to increase the stability of the program.
- Fixed the problem that when the format is converted, the conversion error still displays the problem of completion.
- The functions of "Fix wrong file path" and "Open the location of the playlist" are added to the right-click menu of the playlist in the media library.
- Added the function of remembering the serial number of StatackElement in the user-defined interface.
- Added an option to enable or disable MediaTransControl.
- The "Artist-Album" type is added to the Text element in the custom interface.
- Improvements in other details.
- Other bug fixes.

**Comment by yige233** [comment]
Release [V2.76](https://github.com/zhongyang219/MusicPlayer2/releases/tag/V2.76) 中的两个压缩包（[MusicPlayer2_V2.76_x64.zip](https://github.com/zhongyang219/MusicPlayer2/releases/download/V2.76/MusicPlayer2_V2.76_x64.zip)，[MusicPlayer2_V2.76_x86.zip](https://github.com/zhongyang219/MusicPlayer2/releases/download/V2.76/MusicPlayer2_V2.76_x86.zip)）中的`FAQ.url`和`Help.url`两个快捷方式，指向的均是大佬的另一个项目[TrafficMonitor](https://github.com/zhongyang219/TrafficMonitor)。不知是有意为之还是？

**Reply by zhongyang219** [reply]
> Release [V2.76](https://github.com/zhongyang219/MusicPlayer2/releases/tag/V2.76) 中的两个压缩包（[MusicPlayer2_V2.76_x64.zip](https://github.com/zhongyang219/MusicPlayer2/releases/download/V2.76/MusicPlayer2_V2.76_x64.zip)，[MusicPlayer2_V2.76_x86.zip](https://github.com/zhongyang219/MusicPlayer2/releases/download/V2.76/MusicPlayer2_V2.76_x86.zip)）中的`FAQ.url`和`Help.url`两个快捷方式，指向的均是大佬的另一个项目[TrafficMonitor](https://github.com/zhongyang219/TrafficMonitor)。不知是有意为之还是？

抱歉，是我把链接弄错了，会改正过来
